crates/rules.bzl

```diff
@@ -105,23 +113,27 @@ def _assemble_crate_impl(ctx):
         args.append("--readme-file")
         args.append(ctx.file.readme_file.path)
         inputs.append(ctx.file.readme_file)
+    if ctx.file.license_file:
+        args.append("--license-file")
+        args.append(ctx.file.license_file.path)
+        inputs.append(ctx.file.license_file)
     ctx.actions.run(
-        inputs = inputs + ctx.attr.target[CrateInfo].srcs.to_list() + ctx.attr.target[CrateInfo].compile_data.to_list() + ctx.files.universe_manifests,
-        outputs = [ctx.outputs.crate_package, ctx.outputs.metadata_json],
+        inputs = inputs + ctx.attr.target[CrateInfo].srcs.to_list() + ctx.attr.target[CrateInfo].compile_data.to_list() + ctx.files.universe_manifests + [ctx.file.workspace_refs],
```

Comment:
We fix a bug where `compile_data` was not passed into the assembled crate - building would then fail on the downloader side.